pkg/secretlock/secretlock.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package secretlock
+
+// EncryptRequest for encrypting remote kms requests
+type EncryptRequest struct {
+	Plaintext                   string
+	AdditionalAuthenticatedData string
```

Comment:
minor: suggest rename to AEAD to avoid the mouthful

Reply:
I didn't wanna diverge from struct names in Tink.. they have similar names